packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -101,6 +124,7 @@ class _FlexibleSpaceBarState extends State<FlexibleSpaceBar> {
 
     // title
     if (config.title != null) {
+      final ThemeData theme = Theme.of(context);
```

Comment:
We call this variable themeData about 50% of the time. I like themeData a little better :-)